packages/picasso-lab/src/OverviewBlock/OverviewBlock.tsx

```diff
@@ -49,6 +49,8 @@ export type Props = BaseProps &
     label: string
     /** The color variant  */
     variant?: Variant
+    /** Sets darker color for vertical separator */
+    hasDarkerVerticalSeparator?: boolean
```

Comment:
This name looks long and unclear to me. What do you think about making such a prop?

```suggestion
    separatorVariant?: 'default' | 'dark'
```

Reply:
do we really need 2 types of the separator? It looks like they just asked to change the color. Am I missing something?

Reply:
We don't need, via: [Slack](https://toptal-core.slack.com/archives/CCC3GP6CC/p1627379282026500?thread_ts=1627284635.023100&cid=CCC3GP6CC).